api/db/migrations/trip/20191125100001_create_export_view.up.sql

```diff
@@ -0,0 +1,45 @@
+CREATE MATERIALIZED VIEW trip.export AS (
+  SELECT
+    cpp.operator_id as operator_id,
+    tis.territory_id as start_territory_id,
+    tie.territory_id as end_territory_id,
+    cpp.acquisition_id::varchar as journey_id,
+    cpp.trip_id as trip_id,
+    cpp.datetime as journey_start_datetime,
+    trunc(ST_X(cpp.start_position::geometry)::numeric, cis.density::int) as journey_start_lat,
+    trunc(ST_Y(cpp.start_position::geometry)::numeric, cis.density::int) as journey_start_lon,
+    cpp.start_insee as journey_start_insee,
+    cis.postcodes[0] as journey_start_postalcode,
+    cis.town as journey_start_town,
+    null as journey_start_EPCI, -- TODO
```

Comment:
ça viendra plus tard quand on aura les données d'EPCI dans les territoires
```suggestion
    null as journey_start_epci, -- TODO
```